### ISSUE TITLE
LAM Internal Structure Options 

### ISSUE THREAD

**Issue by snrdg102417** [MegaMek/megameklab#109]
MegMekLab v0.43.4 LAM Internal Structure Options are standard and endo steel. Per Catalyst Game Labs Interstellar Operations p. 114 "Internal Structure: LAMs may not use any internal structure that requires critical hit slots, such as endo steel."

**Comment by HammerGS** [comment]
I'm not seeing this in the coming version.

![image](https://user-images.githubusercontent.com/13752452/33002482-bb9284fa-cd71-11e7-84c5-0a8cad34009d.png)

**Comment by neoancient** [comment]
This was fixed in 3178d06.

**Comment by snrdg102417** [comment]
Thank you HammerGS and neoancient for the replies.

My apologies for not being able to properly show what I am writing about but I hope the line below can be seen.

[InternalStructure.docx](https://github.com/MegaMek/megameklab/files/1488740/InternalStructure.docx)

**Comment by neoancient** [comment]
We understand what the problem is. This has been closed because the fix for the bug has been committed. There has not been a new release since the fix.

**Comment by snrdg102417** [comment]
Afternoon from WA neoancient,

Thank you for the reply and clarification that the issue has been fixed. While I was in the military I took some computer classes and more after I got out. I learned that programming is not one of my talents, however I can do a creditable job at making notes for the steps. Again thank you again for the explanation. I'm a daft old retired submarine sailor that takes a while to catch on. ;-)
